include/fluent-bit/tls/flb_tls_info.h.in

```diff
@@ -0,0 +1,37 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2022 The Fluent Bit Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef FLB_TLS_INFO_H
+#define FLB_TLS_INFO_H
+
+#ifdef FLB_HAVE_TLS
+
+#ifdef FLB_SYSTEM_MACOS
+/* On macOS, default PEM format certificates are not provided by
+ * default. Instead, we'll generate them into:
+ * "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_SYSCONFDIR@/certs/rootcert.pem"
+ * during the building process.
+ */
+#define MACOS_DEFAULT_CA "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_SYSCONFDIR@/certs/rootcert.pem"
```

Comment:
since this is just two macros, please move them to the official flb_info.h.in (by checking also that TLS has been enabled)

Reply:
Noted. I fixed it in https://github.com/fluent/fluent-bit/pull/6035/commits/7558630edc1e0ef3abbd7407da4b21f3d75f486a.